lib/web_ui/dev/integration_tests_manager.dart

```diff
@@ -663,24 +643,10 @@ const Map<String, List<String>> blockedTestsListsMapForModes =
 // TODO(nurhan): Remove the failing test after fixing.
 const Map<String, List<String>> blockedTestsListsMapForRenderBackends =
     <String, List<String>>{
-  'auto': [
-    'image_loading_integration.dart',
-    'platform_messages_integration.dart',
-    'profile_diagnostics_integration.dart',
-    'scroll_wheel_integration.dart',
-    'text_editing_integration.dart',
-    'treeshaking_integration.dart',
-    'url_strategy_integration.dart',
-  ],
```

Comment:
@yjbanov @hterkelsen is this useful to have these? I can also keep this list and not run these tests if we think html+canvaskit is enough?